doc/sources/quick-start.rst

```diff
@@ -14,17 +14,455 @@
 .. * limitations under the License.
 .. *******************************************************************************/
 
-.. _quick_start:
+.. |intelex_repo| replace:: |intelex| repository
+.. _intelex_repo: https://github.com/intel/scikit-learn-intelex
 
-############################################
+####################
 Quick Start
-############################################
+####################
+
+Get ready to elevate your scikit-learn code with |intelex| and experience the benefits of accelerated performance in just a few simple steps. 
+
+Compatibility with Scikit-learn*
+---------------------------------
+
+Intel(R) Extension for Scikit-learn is compatible with the following versions of scikit-learn:
+
+* 1.0.X
+* 1.1.X
+* 1.2.X 
+* 1.3.X
+
+Integrate |intelex|
+--------------------
+
+Patching 
+**********************
+
+Once you install Intel*(R) Extension for Scikit-learn*, you replace algorithms that exist in the scikit-learn package with their optimized versions from the extension. 
+This action is called ``patching``. This is not a permanent change so you can always undo the patching if necessary.
+
+To patch Intel® Extension for Scikit-learn, use one of these methods: 
+
+.. list-table:: 
```

Comment:
Not sure if this is quick start - offering 3 options to start with might be overwhelming - can we just do this really as quick start and offer single simple working version that people can try and reproduce and then start talking about different options? 

on 3 options we probably should highlight usecases then this should be used - like patching itself is just simple way to start playing with it, but non of AutoML maintainers will be using it  - they will use direct imports from sklearnex

Reply:
Do you want to move other options to somewhere else? Maybe Developer Guide and just provide a link from the quick start?

Reply:
Might be just having short version for landing and then have detailed version elsewhere. The pint of quick start is to have simple working version.

So dev guide should work